Orange/tests/test_fss.py

```diff
@@ -74,27 +74,24 @@ def test_defaults(self):
 
 class TestRemoveNaNColumns(unittest.TestCase):
     def test_column_filtering(self):
-        data = Orange.data.Table("iris")
+        data = Table("iris")
```

Comment:
Could make common datasets global for all the test cases to use and not load separately?